pkgs/top-level/all-packages.nix

```diff
@@ -14402,6 +14402,8 @@ with pkgs;
 
     cpupower = callPackage ../os-specific/linux/cpupower { };
 
+    cpuset = callPackage ../os-specific/linux/cpuset { };
+
```

Comment:
Oh actually one more thing I just noticed: This doesn't seem to be a kernel package, so it shouldn't be in the `linuxPackages` attribute set, but rather a top-level attribute. Just move this line to somewhere outside `linuxPackagesFor`. There is a loose order in `all-packages.nix` which you could try to follow too (see very top), but really it's all over the place.